packages/core/upload/server/src/controllers/admin-upload.ts

```diff
@@ -92,6 +92,7 @@ export default {
     const signedFiles = await async.map(uploadedFiles, getService('file').signFileUrls);
 
     ctx.body = await pm.sanitizeOutput(signedFiles, { action: ACTIONS.read });
+    ctx.status = 201;
```

Comment:
It's not, here it just looked cleaner since I'd have to do `...await pm.sanitizeOutput` and we don't seem to have a preference throughout the code base either way if we use the `ctx.responseMethods()`